punctuation.yaml

```diff
@@ -48,7 +48,7 @@ half_shape:
   '''' : { pair: [ '‘', '’' ] }
   '"' : { pair: [ '“', '”' ] }
   '\' : [ 、, '\', ＼ ]
-  '|' : [ ·, '|', ｜, '§', '¦' ]
+  '|' : [ ·, '|', ｜, '§', '¦', '￤' ]
```

Comment:
Why add the full shape symbol under half_shape here?

Reply:
Because there is already the full-width vertical bar `｜`. Most full-width symbols are included in the half_shape section, after their respective narrow forms.

Reply:
I think if there're full shape misplaced as half shape, we should fix it removing them, instead of making the confusion worse.

Reply:
It's not misplaced. Naturally narrow/neutral or half-width forms are intended for latin alphanumerical characters. But CJK should almost always use full-width or naturally wide characters. Mixture use, especially full-width CJK characters with half-width numerals, is the dominant form, and that's what half-shape is originally intended for. Otherwise one should choose ASCII style punctuators.

Reply:
Why is "￤" used with CJK? what's its use case, is it commonly used?

Reply:
Even ¦ is not commonly used in Latin alphabetic languages. Some may use them as delimiters or ornaments. Then full-width forms is preferred in CJK texts.

Reply:
If it's a rare symbol, it should not be here. It can be put with other rare symbols starting with '/'

Reply:
The thing is, the broken bar `¦` is the printed key on IBM and most for-windows keyboards, and that's why it has a full-width version, along with all ASCII characters. But windows has remapped the key to vertical bar `|` while maintaining the keycap as the broken bar `¦`. So, it's weird to remove the broken bar while there are still plenty of spaces.

Reply:
As far as I know, the half shape ¦ is already rearly used, the full shape ￤ is even rearer. Just remove it from this half_shape section please.